cmd/getters.go

```diff
@@ -145,8 +145,21 @@ func isGitRepo() bool {
 }
 
 func getGitStatus() map[string]string {
-	gitBranch, eb := exec.Command("sh", "-c", "git branch --show-current").Output()
-	if eb != nil {
+	branch, eb := os.ReadFile(".git/HEAD")
```

Comment:
while testing I have noticed that this fails in case you are within a sub-folder of the project (since the path `.git/` does not point to the root dir), whereas these cases are handled automatically by the `git branch` parser. We need to understand how to solve this case (probably worth looking into `go-git` for it, at this point).

Reply:
We can walk back up until we hit `.git`, right?

Reply:
Would you like me to leverage `go-git` instead of reinventing the wheel?

Reply:
I'm really not sure `go-git` is a good choice though, looking through its issue tracker leads me to believe it's not being taken care of. git2go seems like a better, slightly lower-level option.